### PR TITLE
Adding Beeceptor

### PR DESCRIPTION
**Pull request by ankitjaininfo**
https://beeceptor.com/

Beeceptor provides Rest API mocking and intercepting with no code. You can upload the OAS and get the mocking server up and running in no time

* Named endpoints
* Interceptor HTTP Request, Mock API calls, Proxy-pass to real APIs.
* Free plan and no sign up required.

Refer to documentation: https://docs.beeceptor.com/docs/features-oas/

### CODE REVIEW

**Comment by darrelmiller** [comment]
@ankitjaininfo We (as in the maintainers of this site) need to have a conversation about commercial SAAS services and whether they are appropriate for this list.

**Comment by daveshanley** [comment]
> @ankitjaininfo We (as in the maintainers of this site) need to have a conversation about commercial SAAS services and whether they are appropriate for this list.

We should do this sooner than later. I think adding a new commercial category and pushing all non-FOSS tools into that category could be a potential solution.

**Comment by ankitjaininfo** [comment]
I think this repo has the biggest collection of tools. I liked the idea of explicitly declaring "Commercial Tools" as a section or another dedicated page, which will help this site become a comprehensive directory.

**Comment by darrelmiller** [comment]
@daveshanley As some SAAS products have free editions, I think we need to qualify that unless the source code for the tool is available in a public source repo with an OSS license, then the tool should be listed in the commercial category.

**Comment by AndrewWalsh** [comment]
@darrelmiller @philsturgeon @daveshanley @mbifulco  Can I clarify our stance on this matter? Is there anything I should be aware of with regard to defining commercial projects in these YAML files? Does this apply even in cases where the commercial offering has an open-source repository?

Beyond this, immediate thoughts on this one, typo with `reuiqred`:

`An HTTP interceptor and rule based mocking service for Rest APIs. No coding reuiqred to create a mock endpoint. No sign-up required.`

Should be:

`An HTTP interceptor and rule based mocking service for Rest APIs. No coding required to create a mock endpoint. No sign-up required.`

Would also be nice to clarify `v3_1: xxx` status. The `github: ` line can be removed given it's empty. I can provide these suggestions in a review, but would appreciate guidance from those pinged before I go down a path leading towards approval.

Beyond this and a few other potential changes, I don't have opposition to the product. Looks viable, though impossible to validate its claims without a paid plan/authorisation. Following its "simulate in web-browser" path leads to an auth wall. Company appears to be active, overall I have no reasons to suspect (most) of its claims.

- https://www.linkedin.com/company/beeceptor

**Comment by philsturgeon** [comment]
Let’s have one last SaaS merged then when we merge new site we can split things up into FOSS/SaaS.

File: _data/tools.yml [hunk 1]
```diff
@@ -907,6 +907,16 @@
     database schema from an OpenAPI description. Based on Yii Framework.
   v3: true
 
+- name: Beeceptor
+  category: mock
+  language: SaaS
+  link: https://beeceptor.com/
+  github:
```
Comment (AndrewWalsh) [inline _data/tools.yml:914]:
```suggestion
```

File: _data/tools.yml [hunk 2]
```diff
@@ -907,6 +907,16 @@
     database schema from an OpenAPI description. Based on Yii Framework.
   v3: true
 
+- name: Beeceptor
+  category: mock
+  language: SaaS
+  link: https://beeceptor.com/
+  github: 
+  description: >
+     "\U0001F41D An HTTP interceptor and rule based mocking service for Rest APIs. No coding reuiqred to create a mock endpoint. No sign-up required."
```
Comment (AndrewWalsh) [inline _data/tools.yml:916]:
```suggestion
     "\U0001F41D An HTTP interceptor and rule-based mocking service for REST APIs. No coding required to create a mock endpoint. No sign-up required."
```

**Comment by AndrewWalsh** [comment]
@philsturgeon Sounds good, I've a few minor changes @ankitjaininfo if you wouldn't mind taking a look. Then we'll get this merged.

**Comment by ankitjaininfo** [comment]
Appreciate all of your consideration and review of this change. I have completed the suggested changes. Thanks.

**Pull request merged**